packages/react-components/src/atoms/Display.tsx

```diff
@@ -6,8 +6,14 @@ interface DisplayProps {
   readonly children: TextChildren;
   readonly styleAsHeading?: keyof typeof headlineStyles;
 }
-const Display: React.FC<DisplayProps> = ({ children, styleAsHeading = 1 }) => (
-  <h1 css={[layoutStyles, headlineStyles[styleAsHeading]]}>{children}</h1>
+const Display: React.FC<DisplayProps> = ({
```

Comment:
I want to keep allowed component usage as low as possible because it my experience otherwise it leads to a hard to understand component structure like on the YLD website. In particular, I've avoided passing arbitrary props to HTML elements, or passing arbitrary styles to components. Can you tell me what is the use case for the extra props on the typography element here, so that we can maybe look for a less invasive solution? What does rehype need to set other than children?

Reply:
It needs to set the id for jumping into the multiple areas. I changed it to be more intentional.
But we need to revisit this :)

Reply:
@fampinheiro do you think I should write a `react-components/README.md` with some principles that the library follows?